src/Interpolations.jl

```diff
@@ -4,6 +4,7 @@ export
     interpolate,
     interpolate!,
     extrapolate,
+    scale,
```

Comment:
5 minutes ago I wrote this method (see, I'm already using it :smile:):
```jl
function GridDeformation{FV<:FixedVector}(u::ScaledInterpolation{FV})
    N = length(FV)
    ndims(u) == N || throw(DimensionMismatch("Dimension $(ndims(u)) incompatible with vectors of length $N"))
    GridDeformation{eltype(FV),N,typeof(u),typeof(u.ranges[1])}(u)
end
```
But of course I can just add the module name, if you don't want to export it.